specification/src/main/asciidoc/webprofile/jpa-cdi/CDI-JPA.adoc

```diff
@@ -0,0 +1,148 @@
+:cdi-spec: https://jakarta.ee/specifications/cdi/4.0
+:jpa-spec: https://jakarta.ee/specifications/persistence/3.0/jakarta-persistence-spec-3.0
+
+=== Jakarta Persistence & Jakarta Context and Dependency Injection (CDI) Integration
+This section specifies the requirements for Jakarta EE environments that integrate both {jpa-spec}[Persistence] and {cdi-spec}[CDI].
+
+
+==== Obtaining an Entity Manager Factory using CDI
+
+A container environment might feature built-in integration of
+Jakarta Persistence with the CDI bean manager, allowing
+injection of a container-managed entity manager factory using
+the annotation _jakarta.inject.Inject_. In the Jakarta EE
+environment, the container must feature built-integration of
+Jakarta Persistence with the CDI bean manager.
+
+For each persistence unit, a container which features such
+built-in integration must make available a bean with:
+
+- bean type _EntityManagerFactory_,
+- the qualifiers specified by _qualifier_ XML elements in
+_persistence.xml_, or _jakarta.enterprise.inject.Default_,
+if no qualifiers are explicitly specified,
+- scope _jakarta.enterprise.context.ApplicationScoped_,
+- bean name given by the name of the persistence unit,
+- no interceptor bindings,
+- a bean implementation which satisfies the requirements of
+this specification for a container-managed entity manager
+factory.
+
+Furthermore, the container must make available five beans with:
+
+- bean types _CriteriaBuilder_, _PersistenceUnitUtil_, _Cache_,
+_SchemaManager_, and _Metamodel_, respectively,
+- the qualifiers specified by _qualifier_ XML elements in
+_persistence.xml_, or _jakarta.enterprise.inject.Default_,
+if no qualifiers are explicitly specified,
+- scope _jakarta.enterprise.context.Dependent_,
+- no interceptor bindings,
+- a bean implementation which simply obtains the instance of
+the bean type by calling the appropriate getter method of
+the _EntityManagerFactory_ bean.
+
+==== Obtaining an Entity Manager Factory using CDI
```

Comment:
This `Obtaining an Entity Manager Factory using CDI` section is a duplicate of same above.

Reply:
One of these passages is meant to be about injecting the `EntitiyManager`. Copy/paste error, I guess.

Reply:
Changed to Injecting...